### PR TITLE
Add contribute.json

### PR DESCRIPTION
**Pull request by jaywink**
This document is required for example for all Mozilla websites.
Might make sense for Riot sites to also describe themselves?

See: https://www.contributejson.org/

Signed-off-by: Jason Robinson <jasonr@matrix.org>

### CODE REVIEW

File: res/contribute.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (turt2live) [inline res/contribute.json]: According to the docs it should be at the project root?

Comment (jaywink) [reply res/contribute.json]: It's copied into `webapp` in the build [here](https://github.com/vector-im/riot-web/pull/12251/files#diff-c1e2b8b85603326af7b040e66f254aefR69) and as such is in the same place as `config.json` if I've understood the build process correctly?

Comment (turt2live) [reply res/contribute.json]: This is more what I'm referencing:
image

Comment (turt2live) [reply res/contribute.json]: the build tools can pull from there too, though it'll also need adding to the `files` in the package.json

Comment (jaywink) [reply res/contribute.json]: > This is more what I'm referencing:
> image

Ah right, you're referring to how it's deployed on riot.im/app for example. True, this wont work on the NV maintained riot web's at all. I guess the idea was not so much the NV installed riot but any hosted riot on a root path of their domain.

Comment (jaywink) [reply res/contribute.json]: > the build tools can pull from there too, though it'll also need adding to the `files` in the package.json

`res` is already in files - will that do? I verified `yarn build` does end up copying the file, but if it's needed explicitly somewhere else can add.

Comment (turt2live) [reply res/contribute.json]: The problem has nothing to do with the build process: I literally mean move this file up one directory.

Comment (jaywink) [reply res/contribute.json]: Sorry, I seriously have had enough coffee but somehow managed to totally misunderstand everything you clearly wrote while juggling other things. Fixed now.

Comment (turt2live) [reply res/contribute.json]: No worries, thanks!

**Pull request merged**